lib/Doctrine/Common/Cache/CacheProvider.php

```diff
@@ -135,6 +136,19 @@ public function save($id, $data, $lifeTime = 0)
     /**
      * {@inheritdoc}
      */
+    public function deleteMultiple(array $keys)
+    {
+        $namespacedKeys = array();
+        foreach ($keys as $key) {
+            $namespacedKeys[] = $this->getNamespacedId($key);
```

Comment:
`return $this->doDeleteMultiple(array_values(array_map([$this, 'getNamespacedId'], $keys)));`